### ISSUE TITLE
First-Time User Popup - Announcement

### ISSUE THREAD

**Issue by Manali-SS** [Paul-Austin-Oswego-CSC480-HCI521/OZ-CSC-480-HCI-521-Spring-2025#108]
Test Title: Verify focus for First-Time User Popup
Reference ID: LOG_061
Setup: Open the web application
Execution Steps: 
Ensure that the popup is announced properly when it appears.
Ensure that the keyboard focus moves to the popup when it appears.
Try using Tab to switch between close, "Sign In with Gmail" and "Continue as Guest" options.
Ensure focus is trapped inside the popup which means focus does not move to the background content.
Expected Results: 
The popup should be fully navigable using the Tab key, with focus trapped inside.
The popup should be properly labeled and announced.

[Video link](https://drive.google.com/file/d/1YvW-D1pBM1xXO6YA7GJMJ9ywuT0twkYO/view?usp=drive_link)

**Comment by Manali-SS** [comment]
@gob1ingo I re-tested this issue. There is still no announcement for the pop-up. Also, the screen reader reads all the components behind the pop-up.
